City-of-Helsinki/berth-reservations-admin
# VEN-322 | Add ContactInformationCard and ActionHistoryCard placeholder components

## Description :sparkles:

* Add ContactInformationCard and ActionHistoryCard placeholder components
* Use shared address formatter

## Issues :bug:

### Closes :no_good_woman:

* [VEN-322](https://helsinkisolutionoffice.atlassian.net/browse/VEN-322): Two placeholder cards on the individual harbor page

## Testing :alembic:

### Automated tests :gear:️

* Added tests for new components and utility
* Updated relevant snapshots and changed test data (split address parts)

### Manual testing :construction_worker_man:

* Check that new components work and are visible:
  * Single harbor page
  * Single winter storage area page
  * Single application page
* Check address formatting:
  * Harbor list, expanded row
  * Single harbor page, harbor card
  * Winter storage area list, expanded row
  * Offer page, harbor card
